functions/Set-DbaDatabaseState.ps1

```diff
@@ -285,9 +279,14 @@ Gets the databases from Get-DbaDatabase, and sets them as SINGLE_USER, dropping
                 Write-Message -Level Warning -Message "Database $db is a system one, skipping"
                 Continue
             }
+            $stateCache = @{}
```

Comment:
Why is this variable called "cache"?

Reply:
because we may get out-of-order instances when piped, and we only need one call per instance.

Reply:
Oh, I understand what the variable is for...just curious to know why it is named in that manner. Meaning we don't have variables in most commands with "cache". Why not just call it `$dbState` like it is in the other command in this PR?

Reply:
it's a cache. $dbState is for a single instance. $dbStates sounded funny. let's settle for $dbStatuses ?